plugin/vcs/gitlab/.gitlab/sql-review.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
```

Comment:
Can we minimize the file and move complexity to our Go code?

We could upgrade the logic to support more features in the future. It’s more difficult to update this file than to upgrade the Go code. We’re hoping this file will never change anymore.

We don’t have permission to read the PR content, do we?